api/HTMLMediaElement.json

```diff
@@ -3659,7 +3659,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLMediaElement/videoTracks",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
This is one of the APIs I set to false in https://github.com/mdn/browser-compat-data/pull/3689. In https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/media/html_media_element.idl?l=84&rcl=e0e92459afd805c944dc3e22a3dc80c9e78fe522 it has `RuntimeEnabled=AudioVideoTracks` and it's not been enabled.